### ISSUE TITLE
 phpBB3: Duplicate entry for key 'username'

### ISSUE THREAD

**Issue by JovanJ** [mybb/merge-system#5]
I just installed a vanilla MyBB 1.6.9 and then tried to merge my phpBB 3.0.10 forum with version 1.6.7 of the merge system. The "Database Configuration" step went fine but then it failed in the "Users" step with this error:

Code:
471 users are importing right now. There are 0 users left to import and 0 pages left.

Loading data from database...
MyBB SQL Error
MyBB has experienced an internal SQL error and cannot continue.

SQL Error:
1062 - Duplicate entry 'LÃ¶se' for key 'username'
Query:
INSERT INTO mybb_users (`usergroup`,`additionalgroups`,`displaygroup`,`import_usergroup`,`import_additi?onalgroups`,`import_displaygroup`,`import_uid`,`username`,`password`,`salt`,`log?inkey`,`email`,`regdate`,`lastactive`,`lastvisit`,`website`,`showsigs`,`signatur?e`,`showavatars`,`timezone`,`avatardimensions`,`avatartype`,`avatar`,`lastpost`,?`icq`,`aim`,`yahoo`,`msn`,`hideemail`,`allownotices`,`regip`,`lastip`,`longregip?`,`longlastip`,`language`,`passwordconvert`,`passwordconverttype`,`postnum`,`inv?isible`,`birthday`,`birthdayprivacy`,`subscriptionmethod`,`receivepms`,`receivef?rombuddy`,`pmnotice`,`pmnotify`,`showquickreply`,`ppp`,`tpp`,`daysprune`,`timefo?rmat`,`dst`,`buddylist`,`ignorelist`,`style`,`away`,`awaydate`,`returndate`,`ref?errer`,`referrals`,`reputation`,`timeonline`,`showcodebuttons`,`totalpms`,`unrea?dpms`,`pmfolders`,`notepad`,`threadmode`,`showredirect`,`dateformat`,`dstcorrect?ion`,`warningpoints`,`moderateposts`,`moderationtime`,`suspendposting`,`suspensi?ontime`,`suspendsignature`,`suspendsigtime`,`coppauser`,`classicpostbit`,`logina?ttempts`,`failedlogin`,`usernotes`) VALUES ('2','','2','931','','931','3397','LÃ¶se','', ... snip ...)

I checked the phpBB users database and it only had one entry with that username. MySQL is version 5.1.68.

The phpBB forum had originally been running with a few mods but they got all dumped over the years (maintenance nightmare, which is why I am looking at MyBB now). We run Joomla with a JFusion phpBB bridge however. phpBB is configured as the master for JFusion.

http://dev.mybb.com/issues/2191

**Comment by DooneW** [comment]
Same error here. Importing from Phpbb3.

**Comment by DooneW** [comment]
Apologies I didn't know if this was too old to reply to so I made a new thread. I want to share an update which might prove helpful.

I had uploaded some plugins, but hadn't activated them or installed them. Another mod who was testing it with me had installed some plugin (I don't know which one). I came the next day to do the merge and ran into the SQL 1062 error. 

The problem was that the plugin had installed an entry on the MySQL table which conflicted with the Merge.

I did a clean install, didn't upload any plugins, didn't active and re-ran the Merge system. It went through flawlessly. The lesson here is that Merge is best used on a clean install, before any configuration or other options are done to the forum. Since there's no documentation or instructions that come with the Merge in the Readme, it might be helpful to warn users up front to use Merge on a fresh install and direct them to NOT activate or install any plugins before then. It could create issues for them.

Thanks a lot for developing this highly valuable tool. If you guys need a hand with documentation I would gladly volunteer some of my time to help and make support a bit easier on you all.
